app/Console/Commands/SyncPeers.php

```diff
@@ -43,20 +43,18 @@ class SyncPeers extends Command
      */
     public function handle(): void
     {
-        Torrent::withoutGlobalScope(ApprovedScope::class)
-            ->leftJoinSub(
-                Peer::query()
-                    ->select('torrent_id')
-                    ->addSelect(DB::raw('SUM(peers.left = 0 AND peers.active = 1) AS updated_seeders'))
-                    ->addSelect(DB::raw('SUM(peers.left != 0 AND peers.active = 1) AS updated_leechers'))
```

Comment:
We can improve this to either `->select(['torrent_id', DB::raw(...), DB::raw(...)])` or `->selectRaw(...)`